### PR TITLE
feat(kuzzle): add ingress resource

### PR DESCRIPTION
**Pull request by Kuruyia**
## What does this PR do ?

This adds the possibility to deploy an `Ingress` ressource directly from the Kuzzle Helm chart.

### CODE REVIEW

**Review by rolljee — APPROVED** [review]

File: charts/kuzzle/Chart.yaml [hunk 1]
```diff
@@ -14,7 +14,7 @@ type: application
 
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
-version: 1.1.0
+version: 1.1.1
```
Comment (rolljee) [inline charts/kuzzle/Chart.yaml:17]: This should be version 1.2.0 since this is a new feature

**Review by rolljee — CHANGES_REQUESTED** [review]

File: charts/kuzzle/Chart.yaml (hunk 1 above)
Comment (Kuruyia) [reply charts/kuzzle/Chart.yaml:17]: If we're going to bump the minor version number then let's also update the version of Kuzzle that is deployed by this chart?

Comment (rolljee) [reply charts/kuzzle/Chart.yaml:17]: Yea, you are right, you mean appVersion: 2.25.0 -> appVersion: 2.28.0 right ?

Comment (Kuruyia) [reply charts/kuzzle/Chart.yaml:17]: Yeah actually we should use the `appVersion` as the tag of the image, so the image that is pulled is the one the user expects by looking at this value.

Comment (rolljee) [reply charts/kuzzle/Chart.yaml:17]: Well good news

* https://github.com/kuzzleio/kuzzle/blob/master/.github/workflows/kuzzle-core.yml
* https://hub.docker.com/layers/kuzzleio/kuzzle/2.28.0/images/sha256-949e7a2bcf37aeb30302e42cec50356a9c2f52be2a95bb3259809ebce4c29cf1?context=repo

Workflow already do that 👍🏼 we can do it now, is it okay for you to update it ? :)

**Pull request merged**